### PR TITLE
Added Updates to Virtuoso Features

### PR DESCRIPTION
**Pull request by danielhmills**
1. Added the following to VirtuosoFeaturesBenefitsByVersion.ttl:

* GraphQL Support
* GeoSPARQL Enhancements
* Encryption at Rest (EAR)

2. Added virtuoso-features-matrix.ttl to the **general** directory

### CODE REVIEW

File: general/VirtuosoFeaturesBenefitsByVersion.ttl [hunk 1]
```diff
@@ -469,3 +469,40 @@ source:
...
+<http://data.openlinksw.com/oplweb/product_feature/graphql-query-endpoint#this>
+  a oplfea:ProductFeature ;
+  wdrs:describedby source: ;
+	skos:altLabel "GraphQL Query Endpoint" ;
+	schema:name "graphql-query-endpoint" ;
+  oplfea:isFeatureOf <http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease80x#this> ;
+	schema:comment """A built-in query service endpoint for handling HTTP messages comprising GraphQL payloads that perform query and (insert and delete) mutation operations from GraphQL clients."""@en;
+  oplben:hasBenefit <http://data.openlinksw.com/oplweb/product_benefit/increases-flexibility-and-interoperability#this>,
+                    <http://data.openlinksw.com/oplweb/product_benefit/EaseofIntegration#this>,
+                    <http://data.openlinksw.com/oplweb/product_benefit/ITInfrastructureInvestmentPreservation#this>.
```
Comment (TallTed) [inline general/VirtuosoFeaturesBenefitsByVersion.ttl]:
```suggestion
   a                  oplfea:ProductFeature ;
   wdrs:describedby   source: ;
   skos:altLabel      "GraphQL Query Endpoint" ;
   schema:name        "graphql-query-endpoint" ;
   oplfea:isFeatureOf <http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease80x#this> ;
   schema:comment     """A built-in query service endpoint for handling HTTP messages comprising GraphQL payloads that perform query and mutation (insert and delete) operations from GraphQL clients."""@en ;
   oplben:hasBenefit  <http://data.openlinksw.com/oplweb/product_benefit/increases-flexibility-and-interoperability#this> ,
                      <http://data.openlinksw.com/oplweb/product_benefit/EaseofIntegration#this> ,
                      <http://data.openlinksw.com/oplweb/product_benefit/ITInfrastructureInvestmentPreservation#this> .
```

File: general/VirtuosoFeaturesBenefitsByVersion.ttl [hunk 2]
```diff
@@ -469,3 +469,40 @@ source:
...
+	schema:comment """âEncryption At Restâ """@en ;
+  oplben:hasBenefit <http://data.openlinksw.com/oplweb/support_benefits#c3cde62562b9bac195fc0f8354fe3d22>.
+
+  <http://data.openlinksw.com/oplweb/product_feature/geosparql-functions#this>
+  a oplfea:ProductFeature ;
+  wdrs:describedby source: ;
+	skos:altLabel "GeoSPARQL Functions" ;
+	schema:name "geosparql-functions" ;
+  oplfea:isFeatureOf <http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease80x#this>;
+  oplben:hasBenefit <http://data.openlinksw.com/oplweb/product_benefit/IndustryStandardsCompliance#this>.
```
Comment (TallTed) [inline general/VirtuosoFeaturesBenefitsByVersion.ttl]:
```suggestion
<http://data.openlinksw.com/oplweb/product_feature/geosparql-functions#this>
   a                oplfea:ProductFeature ;
   wdrs:describedby   source: ;
   skos:altLabel      "GeoSPARQL Functions" ;
   schema:name        "geosparql-functions" ;
   oplfea:isFeatureOf <http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease80x#this> ;
   oplben:hasBenefit  <http://data.openlinksw.com/oplweb/product_benefit/IndustryStandardsCompliance#this> .
```

File: general/VirtuosoFeaturesBenefitsByVersion.ttl [hunk 3]
```diff
@@ -469,3 +469,40 @@ source:
...
+
+  <http://data.openlinksw.com/oplweb/product_feature/virtuoso-server-high-performance#this>
+  a oplfea:ProductFeature ;
+  wdrs:describedby source: ;
+	skos:altLabel "High Performance" ;
+  schema:name "high-performance"
+  oplfea:isFeatureOf <http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease80x#this>,
+                     <http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease70x#this>;
+  rdfs:comment: "Blistering performance - bring on the benchmarks".
```
Comment (TallTed) [inline general/VirtuosoFeaturesBenefitsByVersion.ttl]:
```suggestion
<http://data.openlinksw.com/oplweb/product_feature/virtuoso-server-high-performance#this>
   a                  oplfea:ProductFeature ;
   wdrs:describedby   source: ;
   skos:altLabel      "High Performance" ;
   schema:name        "high-performance" ;
   oplfea:isFeatureOf <http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease80x#this> ,
                      <http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease70x#this> ;
   rdfs:comment       "Blistering performance — bring on the benchmarks!" .
```

File: general/virtuoso-features-matrix.ttl [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+@prefix ns38:	<http://data.openlinksw.com/oplweb/product_feature/data-compression#> .
+@prefix ns39:	<http://data.openlinksw.com/oplweb/product_feature/elastic-cluster#> .
+@prefix ns40:	<http://data.openlinksw.com/oplweb/product_feature/new-extractor-cartridges#> .
+@prefix ns41:	<http://data.openlinksw.com/oplweb/product_feature/new-meta-cartridges#> .
+@prefix ns42:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoReleaseRowStoreClusterCloud64x#> .
+@prefix ns43:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoReleaseRowStoreSingle64x#> .
+@prefix ns44:	<http://data.openlinksw.com/oplweb/product_feature/vectored-execution#> .
+@prefix ns45:	<http://data.openlinksw.com/oplweb/product_feature/LinkedDataPublishing#> .
+@prefix ns46:	<http://data.openlinksw.com/oplweb/product_feature/virtuoso-server-high-performance#> .
+@prefix ns47:	<http://www.openlinksw.com/temp#> .
```
Comment (TallTed) [inline general/virtuoso-features-matrix.ttl:48]: This line should probably be removed, and anything using `ns47:` revised to another prefix/namespace.

File: general/virtuoso-features-matrix.ttl [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns10:this
+	schema:name	"HTTP Compliant Application Server" , "HTTP-Compliant Application Server" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns11:this
+	schema:name	"Industry Standard Query Language Support SPARQL" , "Industry Standard SPARQL Query Language Support" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
```
Comment (TallTed) [inline general/virtuoso-features-matrix.ttl:75]: Why do these (and many others below) have two `schema:name` values? In most (probably all) cases, one is better than the other, in varying ways (better grammar, better pluralization, better punctuation, etc.)...

Also, note that the mix of tabs and spaces makes human editing more challenging. I suggest always using spaces for indents, which many tools can be set to do even when you're typing tabs.

**Review by TallTed — CHANGES_REQUESTED** [review]

File: general/virtuoso-features-matrix.ttl [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+ns32:this
+	schema:name	"Attribute-based Access Controls"^^dt:string ;
+	feat:isFeatureOf	ns33:this ;
+	rdf:type	feat:ProductFeature .
+ns34:this
+	schema:name	"Custom Inference Rules"^^dt:string ;
+	feat:isFeatureOf	ns33:this ;
+	rdf:type	feat:ProductFeature .
+ns35:this
+	schema:name	"Enhanced Relational Mapping"^^dt:string , "Enhanced Relational Mapping" ;
```
Comment (TallTed) [inline general/virtuoso-features-matrix.ttl:161]: `^^dt:string` is rarely helpful. `@en` is likely to be, when we get around to internationalizing the website.
```suggestion
   schema:name      "Attribute-based Access Controls"@en ;
   feat:isFeatureOf ns33:this ;
   rdf:type         feat:ProductFeature .
ns34:this
   schema:name      "Custom Inference Rules"@en ;
   feat:isFeatureOf ns33:this ;
   rdf:type         feat:ProductFeature .
ns35:this
   schema:name      "Enhanced Relational Mapping"@en ;
```

File: general/virtuoso-features-matrix.ttl (hunk 5 above)
Comment (danielhmills) [reply general/virtuoso-features-matrix.ttl:75]: These issues stem from the original output of this document before I made additions. Currently evaluating solutions.

**Pull request merged**